src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,12 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        public static AccessKey CreateFromBase64EncodedAccessKey(string encoded)
+        {
+            var accessKeyStr = Encoding.UTF8.GetString(Convert.FromBase64String(encoded));
+            AccessKey accessKey = JsonConvert.DeserializeObject<AccessKey>(accessKeyStr);
```

Comment:
Another error case is if the access key is a random base64 string and not a base64 encoded string from the dev console. The error returned is `Error: Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: a. Path '', line 0, position 0..`

Maybe a [FormatException](https://docs.microsoft.com/en-us/dotnet/api/system.formatexception?view=net-6.0) could be returned? with an error message describing that the `base64EncodedAccessKey` is not valid

Reply:
Could test this exception yourself with any base64 string like `YXNkYXNkYXNkYXNkYWQ=`

Reply:
i could be more specific by using regex for the base64 string however im not that familiar with how to form regex, but i think for now, i just checked the length of the string (since i noticed that all base64encoded strings have a length of 552)

Reply:
i don't think it's guaranteed to be 552 length. When i tried getting a key myself, i got one that was length 569. could instead surround the `JsonConvert.DeserializeObject` function with a try catch in case there is an error

```
AccessKey accessKey;
try
{
    accessKey = JsonConvert.DeserializeObject<AccessKey>(accessKeyStr);
}
catch (JsonReaderException e)
{
    throw new FormatException($"{nameof(base64EncodedAccessKey)} is not valid", e);
}
```

Reply:
ok done